README.md

```diff
@@ -6,41 +6,27 @@ The following presents two alternatives for configuring your Kafka clients to us
 
 The first alternative is suited for Java clients where you have the ability to modify the client classpath to include the authentication libraries.
 
-The second alternative works with other clients beyond Java, it requires you running a local authentication server.
+The second alternative offers a solution for non-Java Kafka clients, but requires you to set up a local authentication server. This server's role is to securely exchange your application's default credentials with the Kafka client, enabling authentication and authorization for accessing the Kafka cluster.
+
+In either case, your client will use the Google Auth libraries in order to authenticate using the default environment credentials. By default on GCP environments such as GKE or GCE, it means the library will use the environment default credentials. This behavior can be modified to point to different credentials via the GOOGLE_APPLICATION_CREDENTIALS environment variable as described in [this article](https://github.com/googleapis/google-auth-library-java?tab=readme-ov-file#getting-application-default-credentials).
```

Comment:
supernit: replace "GCE, it means the library" with "By default...GCE the library"

Reply:
Re-phrased.